pypit/arflux.py

```diff
@@ -1,4 +1,7 @@
-# Module for fluxing routines
+""" Module for fluxing routines
+"""
+from __future__ import (print_function, absolute_import, division,
```

Comment:
That seems fair... I suppose 1/2=0.5 can always be recast into an int (i.e. `int(1/2) = 0`), if that is desired...